File: LeetCode Problems/1 to 100/26. Remove Duplicates from Sorted Array/26. Remove Duplicates from Sorted Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        set<int> res;
        for (int i = 0; i < nums.size();i++) 
            res.insert(nums[i]);
        nums.clear();
        for(auto it : res)
            nums.push_back(it);
        return nums.size();
    }
};

File: LeetCode Problems/1 to 100/26. Remove Duplicates from Sorted Array/26. Remove Duplicates from Sorted Array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        List<Integer> list = new ArrayList<>();
        for (int i = 1; i < nums.length; i++)
            if (nums[i] != nums[i - 1]) list.add(nums[i - 1]);
        list.add(nums[nums.length - 1]);
        int n = list.size();
        for (int i = 0; i < n; i++) nums[i] = list.get(i);
        return n;
    }
}
